safekeeper/src/timeline.rs

```diff
@@ -485,6 +500,29 @@ impl Timeline {
     }
 }
 
+// pageserver connstr is needed to be able to distinguish between different pageservers
+// it is required to correctly manage callmemaybe subscriptions when more than one pageserver is involved
+// TODO it is better to use some sort of a unique id instead of connection string, see https://github.com/zenithdb/zenith/issues/1105
+fn callmemaybe_connstr(
```

Comment:
If it's not related to the old callmemaybe logic, then I'd like to invent a new name for it. Maybe something like `callback_connstr`, or just `safekeeper_connstr`.

Reply:
I have left this small method as some sort of an easter egg, but I like your sense of naming more.

Yet `callback_connstr` seems too generic for a WAL streaming tasks, so let it be `wal_stream_connection_string`?

Reply:
`wal_stream_connection_string` sounds fine for me